miden-lib/asm/miden/kernels/tx/tx.masm

```diff
@@ -123,49 +123,49 @@ end
 #! Returns the pointer to the note the asset was stored at.
 #! Panics if the non-fungible asset already exists in the note.
 #!
-#! Inputs: [ASSET, note_ptr, num_of_assets]
-#! Outputs: [note_ptr]
+#! Inputs: [ASSET, note_ptr, num_of_assets, note_idx]
```

Comment:
Is the `note_idx` used in this proc? If not, better to remove it.

Reply:
You are right, I just need to store it at the end of the stack. 

Alternatively, I could store it in memory in `export.add_asset_to_note`